ApplicationPattern.yaml

```diff
@@ -2739,10 +2742,6 @@ paths:
                               type: string
                             port-direction:
                               type: string
-                              enum:
-                                - 'core-model-1-4:PORT_DIRECTION_TYPE_MANAGEMENT'
-                                - 'core-model-1-4:PORT_DIRECTION_TYPE_INPUT'
-                                - 'core-model-1-4:PORT_DIRECTION_TYPE_OUTPUT'
                             logical-termination-point:
                               type: string
```

Comment:
line 2821 : ServiceRequestCausesFcPortDeletionRequest : example of fc-uuid to be updated to _ro-2-0-0-op-fc-bm-003_ instead of ro-2-0-0-op-fc-bm-003000

Reply:
Changed as proposed.